frontend/www/js/omegaup/qualitynomination/list.js

```diff
@@ -12,42 +12,55 @@ OmegaUp.on('ready', function() {
     render: function(createElement) {
       return createElement('omegaup-qualitynomination-list', {
         props: {
-          page: payload.page,
+          pages: this.pages,
           length: payload.length,
           myView: payload.myView,
           nominations: this.nominations,
-          totalRows: this.totalRows,
+          pagerItems: this.pagerItems,
+        },
+        on: {
+          'go-to-page': pageNumber => {
```

Comment:
esto funciona si se cambia a

```suggestion
          goToPage: pageNumber => {
```

?

Reply:
Funciona si se cambia donde se manda llamar tambien